migrate/repo_checks.py

```diff
@@ -158,24 +158,32 @@ def fix(self, dry_run=False):
         # Create missing labels
         for label in self.missing_labels:
             if not dry_run:
-                self.api.issues.create_label(
-                    self.org_name,
-                    self.repo_name,
-                    label,
-                    self.labels[label],
-                )
+                try:
+                    self.api.issues.create_label(
+                        self.org_name,
+                        self.repo_name,
+                        label,
+                        self.labels[label],
+                    )
+                except HTTP4xxClientError as e:
```

Comment:
Is it appropriate to link to documentation on the HTTP4XXClientError in a comment? I remember it being annoying to find what exactly was failing and why, and having a link handy next to the code might help in the future.

Reply:
Unfortunately there were no good docs for this to link to which is why it took so long to get this incantation.